data/src/main/java/com/microsoft/azure/kusto/data/KustoColumn.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.azure.kusto.data;
+
+/**
+ * Represents a single Kusto result column with a type, name and nullability information
+ * 
+ * @param <C>
```

Comment:
Is that the only style issue?  It would probably be simpler to just run the code through your formatter before merging. Do you have a formatter you can share?

Reply:
no

Reply:
if its too hard to follow you can leave it and we fix it later